agir/front/components/app/routes.config.js

```diff
@@ -252,6 +252,7 @@ export const routeConfig = {
     hasLayout: true,
     layoutProps: {
       desktopOnlyFooter: false,
+      displayOnMobileApp: false,
```

Comment:
```suggestion
      displayOnMobileApp: true,
```
Vu qu'on veut le cacher partout sauf sur cette page